es/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: news_post
+title: "Liberada la versión 2.4.0-preview1 de Ruby"
+author: "naruse"
+translator: David Padilla
+date: 2016-06-20 18:00:00 +0900
+lang: es
+---
+
+Estamos muy orgullosos de anunciar la liberación de Ruby 2.4.0-preview1.
```

Comment:
This is fine, maybe we can follow the above release notes, `Nos complace anunciar la entrega de Ruby 2.4.0-preview1.`.